op-program/Makefile

```diff
@@ -51,6 +51,11 @@ test:
 verify-sepolia: op-program-host op-program-client
 	env GO111MODULE=on go run ./verify/sepolia/cmd/sepolia.go --l1 $$SEPOLIA_L1URL --l1.beacon $$SEPOLIA_BEACON_URL --l2 $$SEPOLIA_L2URL --datadir /tmp/test-sepolia
 
+verify-devnet:
+	cp ../.devnet/rollup.json ./chainconfig/configs
+	cp ../.devnet/genesis-l2.json ./chainconfig/configs
+	env GO111MODULE=on go run ./verify/devnet/cmd/devnet.go --l1 http://localhost:8545 --l1.beacon http://localhost:5052 --l2 http://localhost:9545 --datadir /tmp/test-devnet
```

Comment:
Thanks for the comment.  I have added the custom config unit test following the suggestion.  For the `verify-devnet item in Makefile, I have removed the copy commands to eliminate the dependency on the devnet setup, but I keep the `go run ...` command so that it will be easier for others to verify `op-program` on devnet.  Let me know if you have further suggestions on this.